settings/settings_manager.py

```
@@ -0,0 +1,33 @@
+CONFIG_PATH = Path.home() / ".image_downloader_config.json"
+
+def load_settings(self):
+    if not CONFIG_PATH.exists():
+        return
+    try:
+        with open(CONFIG_PATH, 'r') as f:
+            extracted_from_load_settings(self, f)
+    except Exception as e:
+        print("Error loading settings:", e)
+
+def extracted_from_load_settings(self, f):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)